includes/class-subscription-lists.php

```diff
@@ -131,19 +135,177 @@ public static function register_post_type() {
+	 * Modify columns on post type table
+	 *
+	 * @param array $columns Registered columns.
+	 * @return array
+	 */
+	public static function posts_columns( $columns ) {  
+		unset( $columns['date'] );
+		unset( $columns['stats'] );
+		$columns['active_providers'] = __( 'Service Providers', 'newspack-newsletters' );
+		return $columns;
+		
+	}
+
+	/**
+	 * Add content to the custom column
+	 *
+	 * @param string $column The current column.
+	 * @param int    $post_id The current post ID.
+	 * @return void
+	 */
+	public static function posts_columns_values( $column, $post_id ) {
+		if ( 'active_providers' === $column ) {
+			$list = new Subscription_List( $post_id );
+			echo esc_html( implode( ', ', $list->get_configured_providers_names() ) );
+		}
+	}
+
+	/**
+	 * Outputs metabox content
+	 *
+	 * @param WP_Post $post The current post.
+	 * @return void
+	 */
+	public static function metabox_content( $post ) {
+		$subscription_list = new Subscription_List( $post );
+		$current_settings  = $subscription_list->get_current_provider_settings();
+		$current_provider  = Newspack_Newsletters::get_service_provider();
+		$empty_message     = '';
+
+		if ( empty( $current_settings ) ) {
+
+			$empty_message = sprintf(
+				// translators: %s is the provider name. Ex: Mailchimp.
+				__( 'This list is not yet configured for %s. Please use the fields below to configure where readers should be added to.' ),
+				'<b>' . esc_html( $current_provider::label( 'name' ) ) . '</b>'
+			);
+
+			$current_settings = [
+				'list' => '',
+				'tag'  => '',
+			];
+		}
+
+		$lists = $current_provider->get_lists();
+
+		wp_nonce_field( 'newspack_newsletters_save_list', 'newspack_newsletters_save_list_nonce' );
+
+		?>
+		<div class="misc-pub-section">
+			<?php if ( ! empty( $empty_message ) ) : ?>
+				<p>
+					<?php echo wp_kses( $empty_message, 'data' ); ?>
+				</p>
+			<?php endif; ?>
+			<label for="newspack_newsletters_list">
+				<?php echo esc_html( $current_provider::label( 'List' ) ); ?>:
+			</label>
+			<select name="newspack_newsletters_list" id="newspack_newsletters_list" style="width: 100%">
+				<option value=""></option>
+				<?php foreach ( $lists as $list ) : ?>
+
+					<option value="<?php echo esc_attr( $list['id'] ); ?>" <?php selected( $current_settings['list'], $list['id'] ); ?> >
+						<?php echo esc_html( $list['name'] ); ?>
+					</option>
+
+				<?php endforeach; ?>
+			</select>
+		</div>
+
+		<div class="misc-pub-section">
+			<label for="newspack_newsletters_tag">
+				<?php esc_html_e( 'Tag', 'newspack-newsletters' ); ?>:
+			</label>
+			<input type="text" name="newspack_newsletters_tag" id="newspack_newsletters_tag" style="width: 100%" value="<?php echo esc_attr( $current_settings['tag'] ); ?>" />
+		</div>
+
+		<?php if ( $subscription_list->has_other_configured_providers() ) : ?>
+			<div class="misc-pub-section">
+				<p>
+					<?php esc_html_e( 'Other providers this List is already configured for:', 'newspack-newsletters' ); ?>
```

Comment:
```suggestion
					<?php esc_html_e( 'Other providers this list is already configured for:', 'newspack-newsletters' ); ?>
```

It feels unnatural to have the list in uppercase in the middle of the sentence. WDYT?